### PR TITLE
[0.3.x] CAL-358 Ensures that resource stream will be properly closed.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Resolves a static analysis finding.

#### Who is reviewing it? 
@glenhein @paouelle 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@bdeining
@rzwiefel

#### How should this be tested?
Full CI run.

#### What are the relevant tickets?

[CAL-358](https://codice.atlassian.net/browse/CAL-358)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java [hunk 1]
```diff
@@ -179,7 +179,9 @@ public BinaryContent getBinaryContent(BufferedImage image)
   public NitfSegmentsFlow getNitfSegmentsFlow(ResourceResponse resourceResponse)
       throws NitfFormatException, IOException {
     notNull(resourceResponse, "resourceResponse must be non-null");
-    return getNitfSegmentsFlow(resourceResponse.getResource().getInputStream());
+    try (final InputStream inputStream = resourceResponse.getResource().getInputStream()) {
```
Comment (paouelle) [inline catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java]: We shouldn't be using try-with-resources on input stream. If an error occurs when closing the stream, that error will bubble out. However, in such situations, there is nothing that can be done anyway and the stream has been read as designed. It is better to handle the stream in a finally close with something like IOUtils.closeQuiertly().
Output streams should always be done via try-with-resources.

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

File: catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java (hunk 1 above)
Comment (bdeining) [reply catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java]: May be out of scope, but is there any concern with `resourceResponse.getResource()` being null?

Comment (coyotesqrl) [reply catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java]: Probably should add a second notnull check.

**Comment by coyotesqrl** [comment]
build now

**Pull request merged**